src/components/Contact.tsx

```diff
@@ -17,22 +22,24 @@ export function ContactForm() {
         Accept: 'application/json',
       },
       data: {
-        name: cName,
-        email: cEmail,
-        message: cMessage,
+        name: getValues('Name').trim().replace(/ {2,}/g, ' ') ?? '', // trim and remove double spaces
+        email: getValues('Email').trim(),
+        message: getValues('Message').trim(),
```

Comment:
i think handle submit can pass the form data into onSubmit, not sure if getValues are required?

ie
```
const onSubmit = (data) => {..
  data: {
    name: data.Name.trim().replace(/ {2,}/g, ' ') ?? '', // trim and remove double spaces
    email: data.Email.trim(),
    message: data.Message.trim(),...
```

Reply:
fixed